docs/modules/ROOT/pages/how-tos/prepare_for_modulesync.adoc

```diff
@@ -0,0 +1,33 @@
+= Make your existing component ready for ModuleSync
+
+* Create `.sync.yml` with the following content:
++
+[source,yaml]
+----
+:global:
+  componentName: my-component <1>
+
+docs/antora.yml:
+  name: my-component <1>
+  title: My Component <2>
+
+LICENSE:
```

Comment:
Do we really need to do that on a per-component basis?

Reply:
Yes if you are asking in general that some parameters need to be defined.

If you are asking specifically about the license year: We are in the understanding that the year in the LICENSE file says from which year the copyright was claimed. It doesn't need to be updated every year. If that assumption is correct, each component should make it clear in which year it was created so that the template engine can pick up that information.
If that is incorrect, we could apply the same license file for every component, always up-to-date with the year.

Reply:
I'm not a license expert, so I'm not really able to answer that one...

Reply:
I propose to remove the custom LICENSE year for now. Components may still override it if they desperately require.

Reply:
Yes, I'm fine with that.